app/assets/stylesheets/styles/components/guides/components/_guide_creation_nav.css.scss

```diff
@@ -0,0 +1,84 @@
+.guide-creation-nav {
+  $guide-nav-color: #5E9546;
+  background: $guide-nav-color;
+
+  // the ul containing the links
+  ul {
+    padding-top: 5px;
+    text-align: center;
+    list-style: none;
+    white-space: nowrap;
+  }
+
+  li {
+    position: relative;
+    display: inline-block;
+  }
+
+  li a {
+    color: $white;
+    font-size: 1.25em;
+    font-weight: 600;
+    vertical-align: middle;
+    color: rgba(255,255,255,0.7);
+    padding: 10px 10px 10px 35px;
+
+    &.disabled {
+      pointer-events: none;
+    }
+
+    &:hover {
+      color: rgba(255,255,255,0.9);
```

Comment:
`0.9` should be written without a leading zero as `.9`<br>Commas in function arguments should be followed by a single space